src/index.tsx

```diff
@@ -1,13 +1,18 @@
 import React from "react";
 import { render } from "react-dom";
 import App from "./App";
-
 import { CssBaseline } from "@mui/material";
 
+// Snackbar/notification stuff
+import { SnackbarProvider } from "notistack";
+import { MaximumSnackbarMessages } from "utils/common/constants";
+
 render(
   <React.StrictMode>
     <CssBaseline />
-    <App />
+    <SnackbarProvider maxSnack={MaximumSnackbarMessages}>
```

Comment:
just be sure you're aware that at this height, any notification will likely cause a full render of the app. doesn't mean it'll be a huge performance impact, just that we should keep it in mind when testing. if a scenario exists where a ton of notifications can be generated, then a lot of renders will happen, re-rendering a bunch of children components.

also to be clear, i'm sure this should be at a very high level so nearly every place in the code can show a notification.

Reply:
It wasn't something I had directly considered so it's good feedback. I hoisted it this high at the suggestion of the original documentation, but I've now added a comment to review the hierarchy at a later stage to determine if it can be lowered.